service/controller/app/v1/resource/chart/desired.go

```diff
@@ -76,3 +77,18 @@ func generateTarballURL(baseURL string, appName string, version string) (string,
 	u.Path = path.Join(u.Path, fmt.Sprintf("%s-%s.tgz", appName, version))
 	return u.String(), nil
 }
+
+func processLabels(projectName string, inputLabels map[string]string) map[string]string {
```

Comment:
Even if it's a private function, I would like to know that what this function does, why it was needed the first place. Probably more godoc? `processLabels` are quite unclear to me.

Reply:
There was already godoc but good point that it needed more detail. Is it clearer now?